language-support/ts/daml-ledger/BUILD.bazel

```
@@ -8,20 +8,19 @@ load("//language-support/ts:defs.bzl", "da_ts_library")
 load("//language-support/ts:eslint.bzl", "eslint_test")
 load("//language-support/ts:typedoc.bzl", "ts_docs")
 load("@build_environment//:configuration.bzl", "sdk_version")
+load("//language-support/ts:jest.bzl", "jest_test")
 
 da_ts_library(
     name = "daml-ledger",
-    srcs = ["index.ts"],
+    srcs = [
+        "index.test.ts",
+        "index.ts",
+    ],
     module_name = "@daml/ledger",
     visibility = ["//visibility:public"],
     deps = [
         "//language-support/ts/daml-types",
-        "@language_support_ts_deps//@mojotech/json-type-validation",
-        "@language_support_ts_deps//@types/ws",
-        "@language_support_ts_deps//cross-fetch",
-        "@language_support_ts_deps//events",
-        "@language_support_ts_deps//isomorphic-ws",
-        "@language_support_ts_deps//ws",
+        "@language_support_ts_deps//:node_modules",
```

Let's add `@types/jest` and `jest-mock-console` explicitly instead of including the whole world. :)
```suggestion
        "@language_support_ts_deps//@mojotech/json-type-validation",
        "@language_support_ts_deps//@types/jest",
        "@language_support_ts_deps//@types/ws",
        "@language_support_ts_deps//cross-fetch",
        "@language_support_ts_deps//events",
        "@language_support_ts_deps//isomorphic-ws",
        "@language_support_ts_deps//jest-mock-console",
        "@language_support_ts_deps//ws",
```